privacyidea/static/templates/documentation.rst

```diff
@@ -36,7 +36,7 @@ System Base Configuration
 -------------------------
 
 {% for k, v in context.systemconfig.items() %}
-{{k.decode("utf-8")}}: **{{v.decode("utf-8")}}**
+{{k}}: **{{v}}**
```

Comment:
I'm not sure about these:
@cornelinux, you explicitly added them in 7a6fef5f7a5b8f79cf1edc82208905776feeb07f to fix encoding issues. But with the ``.decode("utf-8")``, generating the system documentation fails when I use non-ASCII values somewhere (e.g. in the LDAP search filter). 🤔

Reply:
I guess this is so old (12/2016), that it was *before* our LDAP redesign in beginning of 2017.
I remember seeing the system documentation failing in some scenarios but did not drill into this. 
Obviously this is the very reason.

I am fine with your change.